src/open_inwoner/scss/components/SearchResults/SearchResults.scss

```
@@ -28,38 +22,43 @@
     letter-spacing: 0;
     line-height: var(--font-line-height-body);
     text-decoration: none;
-  }
 
-  &__item-title {
-    font-size: var(--utrecht-heading-3-font-size);
-    line-height: var(--utrecht-heading-3-line-height);
-    letter-spacing: 0;
-    margin: 0 0 6px;
-  }
+    .utrecht-heading-3.search-results__item-title {
+      margin: 0 0 var(--spacing-large) 0;
+    }
 
-  &__item-info-container {
-    display: grid;
-    grid-template-columns: 1fr auto;
-    gap: var(--spacing-large);
-    align-items: end;
+    .utrecht-paragraph {
+      margin: 0;
```

```suggestion
      margin: 0;
      max-width: 45rem;
```
When there are no filters the text inside the button becomes really long - longer than the accessible standard ([45-75 characters per line](https://www.accessibility.sa.gov.au/your-role/visual-design/typography)). - 45rem contains a more chars than 75 but it looks and feels better than the full-width texts.

---

45rem in CSS could be a 'magic number' here that comes out of nowhere, so instead I'm going to use an if/else with the render_column/spans template that we have currently, so I can give it the same width as the blue no-results box (and columns on other pages). So it's more consistent.